portfolio/src/main/webapp/index.html

```diff
@@ -6,21 +6,18 @@
     <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <div>
+      <div>
```

Comment:
indentation?

Reply:
done.

Reply:
It still has 4 spaces, but you outline other tags with 2 spaces, could you fix it?

Reply:
i changed all of them to 4, i used the tab.

Reply:
For this div yes, but you have plenty occurrences of 2 spaces in multiple html files. It's better to stick to single rule 2 or 4 spaces (tab) and apply it to all your files. Could you work on this please? There are some free formatter tools which automatically fix such issues, like https://www.freeformatter.com/html-formatter.html

Reply:
done.